Automattic/jetpack
# Google Analytics: delete feature removal notices

## Proposed changes:
Remove Google Analytics deprecation/removal notices introduced in #38078 and #38701.

### Other information:


## Jetpack product discussion
https://github.com/Automattic/vulcan/issues/457

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. If testing locally, rebuild Jetpack.
2. Visit Jetpack Dashboard and My Jetpack, confirm they display correctly.
